pkg/cmd/pulumi/doc.go

```diff
@@ -159,32 +163,34 @@ func findDocstring(pointer, lang string, helper codegen.DocLanguageHelper) (stri
 		object = member
 	}
 
+	var docstring string
+	err = nil
 	switch object := object.(type) {
 	case *schema.Property:
-		docstring, err := genPropertyDocstring(object, pkg, lang, helper)
-		return docstring, true, err
+		docstring, err = genPropertyDocstring(object, pkg, lang, helper)
```

Comment:
Nit: in general I prefer the early return style if there's no further processing of the values being set.